judo-tatami-jsl-jsl2ui/src/main/epsilon/transformations/ui/modules/view/viewLinkDeclarationViewPage.etl

```diff
@@ -0,0 +1,213 @@
+rule ViewLinkPageDefinition
+    transform s: JSL!UIViewLinkDeclaration
+    to t: UI!ui::PageDefinition {
+        guard: rootMenu.containsVisualElement(s)
+
+        t.setId(rootMenu.name + "/(jsl/" + s.getId() + ")/ViewLinkPageDefinition");
+        t.name = s.getFqName() + "::ViewPage";
+        t.container = s.referenceType.equivalent("ViewPageContainer");
+
+        rootMenu.equivalent("Application").pages.add(t);
+
+        t.dataElement = s.transferRelation.target.equivalent("RelationType");
+
+        /*
+        log.info("===================");
+        log.info(t.name + ":");
+        log.info(relations.collect(r | r.name).concat(", "));
+        log.info("===================");
+        */
+
```

Comment:
Is it reserved for latrer use?

Reply:
Used for occasional debugging, once we have everything in place, it'll be removed.